### DISCUSSION TITLE
Where to contribute docs?

### DISCUSSION THREAD

**Discussion by iacore** [GraphiteEditor/Graphite#945] [Help] [answered]
I want to write docs on how to build the tauri desktop app, but I don't know how/where to contribute.

There is a folder called `docs/` in this repo. There is also https://graphite.rs/contribute/, which I have no idea how to edit.

**Comment by Keavon** [comment, marked as answer]
Good question! `/docs` is out of date and will need a major revamp/replacement. Adding to the contribute page would be the right plan. You can find that in `/website/content/contribute/_index.md`. I'll have to manually deploy the update to the website once you land a PR.
